gpkit/interactive/rendering.py

```diff
@@ -0,0 +1,35 @@
+"""Rendering methods"""
+
+"""Save a set of variables to a .csm, for usage
```

Comment:
Docstring should be on the line below function declaration, with the following formatting:

```
"""Save a set of variables to a .csm, for usage with the Engineering Sketch Pad rendering software.
 
Arguments
---------
filepath : string
    the full filepath to the .csm, for example "design.csm" or "renders/version1.csm"

sol : SolutionArray
 
paramdict : dict
    {design parameter string in ESP: corresponding Variable}
 """
```